x-pack/plugin/src/test/resources/rest-api-spec/api/ml.find_file_structure.json

```diff
@@ -5,6 +5,10 @@
       "description":"Finds the structure of a text file. The text file must contain data that is suitable to be ingested into Elasticsearch."
     },
     "stability":"experimental",
+    "headers":{
+      "accept": [ "application/json"],
+      "content_type": ["application/x-ndjson"]
```

Comment:
Since this technically supports sending `application/json` too, should we add it after the preferred `application/x-ndjson`? I can see an argument against not including it as even though the `content-type` header is `application/json`, the request body format must be newline delimited JSON